spec/models/condition_spec.rb

```diff
@@ -76,6 +76,12 @@
         expect(Condition.subst(expr, @vm)).to eq('"0"')
       end
     end
+
+    it "does not change the scope for Tags" do
+      expr = "<value ref=tag, type=text>/virtual/name</value> == \"/managed/foo\""
+      tag = FactoryGirl.create(:tag, :name => "/managed/foo")
+      expect { described_class.subst(expr, tag) }.not_to change { tag.reload.taggings.scope_for_create }
```

Comment:
@kbrock this feels like I'm testing a private API here - do you know if that is so? Could there be a better way to test this?